kakarot_scripts/deployment/dualvm_token_deployments.py

```diff
@@ -134,12 +137,13 @@ async def deploy_dualvm_tokens() -> None:
             "CairoPrecompiles", "DualVmToken", evm_deployments[token["name"]]["address"]
         )
         assert await token_contract.starknetToken() == l2_token_address
-        assert await token_contract.name() == token["name"]
-        assert await token_contract.symbol() == token["symbol"]
+        assert (await token_contract.name()).lstrip("\x00") == token["name"]
```

Comment:
create issue to strip zeroes for zeroes?

Reply:
?

Reply:
to strip zeroes for the name* sorry

or we don't care if the result is 0x00000000name?

Reply:
you mean to remove this `lstrip` ? So an issue to update the sol files? I think that we don't care for now, do we?

Reply:
i'm just worried something like STRK ends up showing like 000000000000000000STRK in the wallets. I actually don't know what it looks like until we can import it and see